helix-view/src/editor.rs

```diff
@@ -65,6 +65,13 @@ where
     )
 }
 
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+pub enum FilePickerLayoutDirection {
+    Horizontal,
+    Vertical,
+}
+
```

Comment:
There are multiple different picker's is this something that should be generalized for all of them?

Reply:
That would probably make sense, yes.

But where to put the setting then? Maybe have a "prefered-layout-direction" in `[editor]`?

Reply:
I would think a new `picker` section separate from the `file-picker` options. The other options really are all specific to the file picker, but I could see other options being desired for all pickers in the future.

Reply:
I did not see any other editors yet with a preview, so the other pickers will not be effected anyway. So maybe we should leave this as is for the time being.

Reply:
Coming back to @EpocSquadron: Maybe I should also change this to `VSplit` and `HSplit` and adapt as suggested below?

If we really open up an entire section for this: Maybe I should then add weights for picker/preview so that the ration of the space used for the two elements can be set, too. You did not like my magic "3" to get the preview twice the size as the picker after all and this is something people tend to be passionate about :-)

Reply:
OK, the configuration is moved. I also added weights to influence how much space each of the two widgets will get.

Reply:
I think it's okay to be opinionated as well. Helix as an editor is already very opinionated as to what type of editing it supports.

If you think the weights adds value go ahead, otherwise I think it's perfectly okay to just set them to something sensible.

> If we really open up an entire section for this

This is also a really good point, do we want an entire new section for changing this layout? If the file picker is the only picker where this is relevant at the moment maybe it's better to keep it simple? I don't how @archseer feels about breaking changes to the config file if this is first implemented specifically for the file picker and later moved to its own section.

Reply:
This should also effect the buffer picker, no?

Reply:
Just checked and it definitely is affecting buffer picker, jumplist picker, diagnostics picker, symbol picker, etc. This is good, and makes my case about it deserving its own configuration section.